packages/dds/sequence/src/sequenceDeltaEvent.ts

```diff
@@ -136,14 +136,14 @@ export abstract class SequenceEventClass<
 	/**
 	 * The first of the modified ranges.
 	 */
-	public get first(): Readonly<ISequenceDeltaRange<TOperation>> {
+	public get first(): Readonly<ISequenceDeltaRange<TOperation> | undefined> {
```

Comment:
```suggestion
	public get first(): Readonly<ISequenceDeltaRange<TOperation>> | undefined {
```
for consistency with above.